### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by lambert3**

### CODE REVIEW

File: src/Components/loadingScreen.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
```
Comment (RonKbS) [inline src/Components/loadingScreen.js:1]: @lambert3 , this should be in a separate branch. If this PR is related to temperate variation charts, then the loading feature Paul was talking about should be in a new branch created off of master

**Review by RonKbS — COMMENTED** [review]
This looks cleaner @lambert3 . Good jog working on it.

Regarding the loading feature, create a separate PR for it. And create a new branch off of master. And  not the branches you've already pushed here. There should be not more than two or three files with changes when it's done properly. Make sure to run `git checkout master` and `git pull origin master` before creating the branch

File: src/Components/DD1 (2).json [hunk 2]
```diff
@@ -0,0 +1,165 @@
+{
```
Comment (RonKbS) [inline src/Components/DD1 (2).json:1]: Does this file have a use, @lambert3 ?

File: src/Components/pieChart.js [hunk 3]
```diff
@@ -9,25 +10,77 @@ class PieChartComponent extends Component {
...
 
     this.state = {
-      labels: this.props.updatePieChartIndicators,
       shouldRedraw: false,
     };
   }
-  
-  render() {
+
+  render(props) {
```
Comment (RonKbS) [inline src/Components/pieChart.js:17]: Is there a reason for passing in props to the render method at this point?

Comment (lambert3) [reply src/Components/pieChart.js:17]: @RonKbS  ,
i will rectify this

File: src/Components/DD1 (2).json (hunk 2 above)
Comment (lambert3) [reply src/Components/DD1 (2).json:1]: yes it  does, its being used by the loading screen component

File: src/Components/loadingScreen.js (hunk 1 above)
Comment (lambert3) [reply src/Components/loadingScreen.js:1]: thanks @RonKbS , let me implement this right away

**Comment by RonKbS** [comment]
@lambert3 , the changes for this PR and https://github.com/Geo-Gecko/ASRT_GG/pull/12 have been merged in using this -> https://github.com/Geo-Gecko/ASRT_GG/pull/16 .

I'm going to be deleting the two branches off of which the above mentioned PRs are based. So please make sure to use new branches if you're working on anything new.

**Pull request closed without merging**